.gitlab-ci.yml

```diff
@@ -558,14 +558,29 @@ gcp.sh:
   variables:
     SCRIPT: gcp.sh
 
-vmware.sh:
+vmware.sh_vmdk:
   extends: .integration_rhel
   rules:
     # Run only on x86_64
     - !reference [.upstream_rules_x86_64, rules]
     - !reference [.nightly_rules_x86_64, rules]
   variables:
-    SCRIPT: vmware.sh
+    SCRIPT: vmware.sh vmdk
+
+vmware.sh_ova:
+  stage: test
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/vmware.sh ova
```

Comment:
You should instead inherit from `.integration_rhel` and define the `SCRIPT` variable, as it is done in the case above. It may be also enough to inherit directly from `vmware.sh_vmdk` job and redefine the `SCRIPT` variable.

Reply:
Yea but I don't want to run it against all runners ._.

Reply:
You can still have that with inheritance. Any field that you specify explicitly will be overridden.

Reply:
I'll run it against all rhel rnuners, if we also want this onprem